api/src/services/agencies_v1/get_agencies.py

```diff
@@ -35,6 +37,20 @@ def get_agencies(
         .where(Agency.is_test_agency.isnot(True))
     )
 
+    if active:
+        stmt = (
+            select(Agency)
+            .join(Opportunity, onclause=Agency.agency_code == Opportunity.agency_code)
+            .join(CurrentOpportunitySummary)
+            .where(Agency.is_test_agency.isnot(True))  # Exclude test agencies
+            .where(
+                CurrentOpportunitySummary.opportunity_status.in_(
+                    [OpportunityStatus.FORECASTED, OpportunityStatus.POSTED]
+                )
+            )
+            .options(joinedload(Agency.top_level_agency), joinedload("*"))
+        )
```

Comment:
Don't create a new stmt, extend the one that already exists. That way any default rules we want for filters don't need to be implemented twice.
```suggestion
    if active:
        stmt = (
            stmt
            .join(Opportunity, onclause=Agency.agency_code == Opportunity.agency_code)
            .join(CurrentOpportunitySummary)
            .where(
                CurrentOpportunitySummary.opportunity_status.in_(
                    [OpportunityStatus.FORECASTED, OpportunityStatus.POSTED]
                )
            )
        )
```

I'm also not certain, but it might be worth considering a query like this instead (no idea which is better performing, could run some `EXPLAIN ANALYZE <query>` in dev/staging to test)

```sql
SELECT
	*
FROM
	api.agency a
WHERE
	is_test_agency IS FALSE
	AND a.agency_code in(
		SELECT
			agency_code FROM api.opportunity o
			JOIN api.current_opportunity_summary cos ON o.opportunity_id = cos.opportunity_id
		WHERE
			opportunity_summary_id in(1, 2))
```

Reply:
adapted new query